modules/site-health/admin-site-health.php

```diff
@@ -29,6 +29,15 @@ class PLL_Admin_Site_Health {
 	 */
 	protected $static_pages;
 
+	/**
+	 * Is SimpleXML PHP extension loaded ?
+	 *
+	 * @since 3.2
+	 *
+	 * @var bool
+	 */
+	protected $simplexml;
```

Comment:
You don't need to make a property. I don't believe that calling `extension_loaded()` several times is an issue.